src/book/book.rs

```diff
@@ -77,10 +77,10 @@ fn create_missing(src_dir: &Path, summary: &Summary) -> Result<()> {
 /// [`iter()`]: #method.iter
 /// [`for_each_mut()`]: #method.for_each_mut
 #[derive(Debug, Clone, Default, PartialEq, Serialize, Deserialize)]
+#[non_exhaustive]
```

Comment:
This is a breaking change and cannot be done in a point release.

Reply:
I'll revert it, then; but, should the lint be kept so we can remember for the next major release, or should it be suppressed?

Reply:
I filed #1835 to track considering it in a point release. I don't have much of an opinion on allowing it, as I don't use clippy. I'm not particularly happy with lots of `allow(clippy)` stuff, but if you'd like to add it, that would be fine.